src/components/pipeline/AWSFilePreview.astro

```diff
@@ -28,10 +28,15 @@ const hasPreview = [
     'gif',
     'svg',
 ].includes(extension);
-const tooBig = extension !== 'html' && file.size > 10000000;
+let tooBig = extension !== 'html' && file.size > 10000000;
```

Comment:
```suggestion
let tooBig =  file.size > 5500000;
```
Why not simply like this (because we will probably have the same problem with other file types?

Then we don't need the exception below.

Reply:
The approach chosen was based on the false? assumption that it would only fail when the `<iframe>` component gets rendered, while fetching larger `content` asynchronously would still succeed. 

Since you had already written an elaborate `const tooBig = extension !== 'html' && file.size > 10000000` if-clause for fetching content, I presumed this part of the code was already well tested with files up to this size.

For now, I agree that simply lowering the limit will be the most straightforward solution. If we feel like overengineering at some point, using readable streams and converting chunks from them to blobs would technically allow handling bigger data in the browser.

Reply:
Nah, I just guesstimated based on rendering time (didn't run into that error locally). Let's just lover the limit and have the same for all and we should be good for now. Maybe when we see too many Multiqc reports bigger than that size we can start looking for alternative approaches.

Reply:
Convincing argument! I agree.